### ISSUE TITLE
Add support for videos

### ISSUE THREAD

**Issue by mvysny** [mvysny/photocloud-frame-slideshow#1]
I'm not sure whether this is possible, but maybe add support for playing videos. Probably the video will not be cached and must be played directly from the stream - it is not viable to download large videos first.

**Comment by rgouveia-github** [comment]
:+1: 
I definitely use this as I have a few videos :-)

**Comment by mvysny** [comment]
I have to check what the video playing options on Android are. Last time I tried to play a mp3 from a stream, the Android MediaPlayer responded with the following error: "what 1 extra 5". Why thank you for such a non-informative retarded error message.

**Comment by cianoz** [comment]
Me too would like to have video support. Hope the dev will work soon on this

**Comment by luislobo** [comment]
Would appreciate this feature but not a break dealer

**Comment by ruiiiur** [comment]
Hello

Any news about this feature. It would be awesome, just bought your app, it’s great, but would be Super with video support. 

Best regards.

**Comment by ruiiiur** [comment]
Helllllloooo

Any update on this possible great improvement for this already great app ?

Thanks ...

RuiiiuR

**Comment by mvysny** [comment]
I think for starters the most basic support for video could be added, with the following limitations:

1. The support for videos will be experimental for the time being, and you will need to enable this explicitly in PhotoCloud Settings.
2. No offline support - videos will always be played from the cloud storage
3. No in/out animations, no pinch-to-zoom for videos, no history scrollback
4. Only a subset of streams will be supported (namely only those which allows downloading content from an InputStream)

**Comment by luislobo** [comment]
I agree with that, even if it’s just with the option to open on external viewer (VLC, for example)

**Comment by mvysny** [comment]
Animated gifs could be treated the same way as videos.

**Comment by TheOriginalMrWolf** [comment]
*BUMP*!! 👍 

The way you proposed to do it in May would be perfect.  I did something similar on my Raspberry Pi photoframe where I would obtain a video URL from whichever cloud service & pass that through to omxplayer which would then play it directly.

In order to get a test feature going quickly, could you do something similar with, for instance, mxplayer (or equivalent)?

Cheers!

**Comment by egorlins** [comment]
Yeah! a lot of videos from events can be also shown. Super feature can be
